applications/object_detection_torch/README.md

```diff
@@ -83,4 +83,19 @@ export LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:/opt/hpcx/ompi/lib"
 
 To containerize the application using [Holoscan CLI](https://docs.nvidia.com/holoscan/sdk-user-guide/cli/cli.html), first build the application using `./holohub install object_detection_torch`, run the `package-app.sh` script and then follow the generated output to package and run the application.
 
-Refer to the [Packaging Holoscan Applications](https://docs.nvidia.com/holoscan/sdk-user-guide/holoscan_packager.html) section of the [Holoscan User Guide](https://docs.nvidia.com/holoscan/sdk-user-guide/) to learn more about installing the Holoscan CLI or packaging your application using Holoscan CLI.
\ No newline at end of file
+Refer to the [Packaging Holoscan Applications](https://docs.nvidia.com/holoscan/sdk-user-guide/holoscan_packager.html) section of the [Holoscan User Guide](https://docs.nvidia.com/holoscan/sdk-user-guide/) to learn more about installing the Holoscan CLI or packaging your application using Holoscan CLI.
+
+## Known Issues
+
+**Limited platform support**: The Faster R-CNN PyTorch model in this example relies on symbols available only in `torchvision<=0.23.0`. arm64 platforms and x86_64 platforms with CUDA 13 drivers (>=R580) are currently not supported as no compatible combination of Holoscan SDK, PyTorch (CUDA), and torchvision (CUDA) are available for these platforms.
+
+At the time of writing, the supported configuration based on PyTorch components availability is as follows:
+- x86_64 platform
+- Latest Holoscan SDK (>=v3.9.0)
+- PyTorch 2.8.0 with CUDA 12.9 support
+- torchvision 0.23.0 with CUDA 12.9 support
```

Comment:
what is missing for JP6? it should have torch 2.8 and torchvision 0.23 with cuda 12.x

Reply:
Thanks for catching that, will update 👍

Reply:
Confirming iGPU support:
```bash
h$ xvfb-run -a ctest -CRelease -R object_detection_torch    
Test project /workspace/holohub/build/object_detection_torch
    Start 1: object_detection_torch_test
1/2 Test #1: object_detection_torch_test ..........   Passed   12.06 sec
    Start 2: object_detection_torch_render_test
2/2 Test #2: object_detection_torch_render_test ...   Passed   15.05 sec

100% tests passed, 0 tests failed out of 2
```